app/config/FrontendAppConfig.scala

```diff
@@ -23,12 +23,16 @@ import play.api.i18n.Lang
 import play.api.mvc.Call
 
 @Singleton
-class FrontendAppConfig @Inject() (configuration: Configuration) {
+class FrontendAppConfig @Inject()(configuration: Configuration) {
 
   val contactFormServiceIdentifier = "EEWFH"
 
+  val platformHost: Option[String] = configuration.getOptional[String]("platform.frontend.host")
+
+  val preferencesFrontendHost: String =
```

Comment:
I think this was fine before 
`val preferencesFrontendHost: String = configuration.get[Service]("microservice.services.preferences-frontend").baseUrl` as that would of picked up the value `preferences-frontend.public.mdtp` from app-config-base which would be fine in a connector call

Reply:
I have a feeling this was meant for **pertaxFrontendHost** instead of **preferencesFrontendHost**

Reply:
Have a look now